webapp/sources/rudder/rudder-web/src/main/elm/build-app.sh

```diff
@@ -8,6 +8,8 @@ ELM_DIR="$( cd "$( dirname "$0" )" && pwd )"
 PROJECTS=("notifications")
 for PROJECT in ${PROJECTS}; do
   cd ${ELM_DIR}/${PROJECT}
+  # To make sure we build up-to-date stuff and avoid corruption
+  rm -rf elm-stuff
```

Comment:
I don't want to download everything at each build when developing it's ok to me until 7.2, and in 7.2 don't do it in dev build

Reply:
Yes I will do it only for release builds starting from 7.2 :+1: